src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -202,42 +199,98 @@ public int getDefaultCloudIndexPrefetchBufferSize() {
      */
     @Override
     public void onStartup() {
+        assertVariantPathsOrSampleNameFileWasSpecified();
+        initializeHeaderAndSampleMappings();
+        initializeIntervals();
+        super.onStartup();
+    }
 
-        List<VCFHeader> headers = new ArrayList<>(variantPaths.size());
-
-        for (String variantPath : variantPaths) {
-
-            final AbstractFeatureReader<VariantContext, LineIterator> reader = getReaderFromVCFUri(variantPath);
-            VCFHeader header = (VCFHeader)reader.getHeader();
+    private void assertVariantPathsOrSampleNameFileWasSpecified(){
```

Comment:
it is unfortunate.  I filed a bug https://github.com/broadinstitute/barclay/issues/52